runtime/oti/j9cfg_builder.h

```diff
@@ -95,8 +95,8 @@
 #define J9_JIT_DATA_CACHE_SIZE (8 * 1024 * 1024)
 #endif /* J9VM_ARCH_X86 && !J9VM_ENV_DATA64 */
 
-#if defined(J9ZOS390) && defined(J9VM_GC_COMPRESSED_POINTERS)
-#define J9_OS_STACK_SIZE (384 * 1024)
+#if defined(J9ZOS390) && defined(J9VM_ENV_DATA64)
+#define J9_OS_STACK_SIZE (1024 * 1024)
```

Comment:
Please add a comment to this to indicate why the VM defaults to a 1MB stack

Reply:
Comment added